elyra/pipeline/tests/test_processor_kfp.py

```diff
@@ -212,12 +212,17 @@ def test_processing_url_runtime_specific_component(monkeypatch, processor, sampl
     processor._component_registry._cached_components = [component]
 
     # Construct hypothetical operation for component
+    operation_name = "Filter text test"
```

Comment:
I see `operation_name` includes a " test" suffix yet the corresponding yaml doesn't include such a string.  Is this `operation_name` equated to anything in the component.yaml or does this value just get rendered as `pipelines.kubeflow.org/task_display_name` below?

Reply:
Good question. `operation_name` corresponds to what would be the `label` field in an equivalent pipeline JSON. So you're right that it's not in the YAML. In that way, I guess these two tests sort of confirm a handful of things about the output of the `_cc_pipeline()` function (and after pipeline compilation):

- `label` is assigned properly by `set_display_name()
- the YAML is properly loaded in `load_component()` and the spec location is therefore correct
- the corresponding `ContainerOp` object is created properly by the factory function and the component parameters passed to it are as expected